core/src/main/java/com/microsoft/applicationinsights/internal/util/DateTimeUtils.java

```diff
@@ -60,7 +60,7 @@ public static long getDateDiff(Date laterDate, Date date, TimeUnit timeUnit) {
      * @throws java.text.ParseException Thrown when failed to parse the given string.
      */
     public static Date parseRoundTripDateString(String roundTripString) throws ParseException {
```

Comment:
What's the difference between `parseRoundTripDateString` and `formatAsRoundTripDate`?
Do we really need both these methods?
If we do please at least refactor